ui/app/components/clients/chart-container.hbs

```diff
@@ -0,0 +1,44 @@
+{{! HBS display template for rendering title, description and stat boxes with a chart on the right }}
+
+<div class="chart-wrapper single-chart-grid" ...attributes>
```

Comment:
Provides layout for these 
<img width="1061" alt="Screenshot 2024-01-23 at 2 19 13 PM" src="https://github.com/hashicorp/vault/assets/68122737/a5fa1ca5-fb59-43a2-b465-3fbadca336a4">

<img width="1069" alt="Screenshot 2024-01-23 at 2 19 25 PM" src="https://github.com/hashicorp/vault/assets/68122737/5dc69b09-f4e1-433f-9b37-13adf830d423">

Reply:
<img width="1070" alt="Screenshot 2024-01-23 at 1 42 54 PM" src="https://github.com/hashicorp/vault/assets/68122737/3e8937ef-6e98-4abd-8179-e67ba2053aa2">